### PR TITLE
Added FlowPlayer component & implemented in video blocks.

### PR DESCRIPTION
**Pull request by benjaminnaesen**

### CODE REVIEW

**Comment by bertyhell** [comment]
title seems to be messed-up in fullscreen: 
image

**Comment by bertyhell** [comment]
either remove the flowplayer tests file or add tests. otherwise the test suite will fail:

image

**Comment by benjaminnaesen** [comment]
image

File: src/components/FlowPlayer/FlowPlayer.tsx [hunk 1]
```diff
@@ -34,9 +35,22 @@ export const FlowPlayer: FunctionComponent<FlowPlayerProps> = ({
...
 	const videoContainerRef = useRef(null);
 	const videoPlayerRef: MutableRefObject<any | undefined> = useRef<any>();
+	const [lastSeekTime, setLastSeekTime] = useState<number | undefined>(undefined);
+
+	useEffect(() => {
+		if (typeof seekTime !== 'undefined' && seekTime !== lastSeekTime && seekTime !== 0) {
+			setLastSeekTime(seekTime);
+			// seekTime should be updated
+			if (videoPlayerRef.current) {
+				console.log('seekTime updated', seekTime);
```
Comment (bertyhell) [inline src/components/FlowPlayer/FlowPlayer.tsx]: console log 😬

**Pull request merged**